.github/workflows/set_rook_ceph_cluster.yaml

```diff
@@ -0,0 +1,52 @@
+name: rook-ceph-cluster
```

Comment:
make this as just ceph cluster? and also update the file name to be `setup_ceph_cluster?`

Reply:
done